Tema1/Animals/Animals/Animals/Cat.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Animals.Animals
+{
+    public class Cat : IAnimal
+    {
+        public void makeSound()
+        {
+            Console.WriteLine("Cat makes sound: Wiwiwiwi");

```

Comment:
Console.WriteLine("This is not a Cat! My PM asked me to transform it into a crocodile ..... makes sound: Wiwiwiwi");

Reply:
o clasa Cat este intotdeauna o pisica; poti folosi alte metode pentru a lua numele clasei